### DISCUSSION TITLE
Foundry installation error

### DISCUSSION THREAD

**Discussion by R3FL3X99** [Cyfrin/foundry-full-course-cu#1664] [Q&A] [answered]
hey guys I'm trying to install foundry but it stops loading but the timer keeps up and it throws this  error

```
curl: (28) Operation timed out after 300231 milliseconds with 0 out of 0 bytes received
```
I'm not using any vpn and I've triple checked my connection it just won't install and I cant start the course locall without it, please help
![Screenshot (120)](https://github.com/Cyfrin/foundry-full-course-f23/assets/99340137/9d3cdb25-2c09-4704-838d-3b1175076aa2)

**Comment by Kshitij-Shringi** [comment, marked as answer]
Check the server status to make sure it is responding.
Try restarting your network connection.
Increase the cURL timeout value.
Disable any firewalls or security plugins that may be blocking the cURL request.
Contact your hosting provider for assistance.

**Reply by R3FL3X99** [reply]
disabled firewall and it worked, thank you so much

**Reply by EngrPips** [reply]
Excellent. Glad you get it to work.

**Comment by EngrPips** [comment]
Hello @R3FL3X99, Please try all of the suggested solutions by @Kshitij-Shringi. Let us know if any of those suggestions help solve the issue.
